app/client-app/src/components/shared/SurveyPage.jsx

```diff
@@ -168,17 +173,24 @@ const SurveyPage = ({
           </div>
 
           <Stack spacing={3} direction="row" align="center">
-            <Button
-              size="lg"
-              colorScheme="red"
-              variant="outline"
-              borderWidth="2px"
-              onClick={clearResult}
-              mr={2}
+            <Tooltip
+              label="This page requires a response before proceeding, so it cannot be cleared. You can continue to change your response before submitting."
+              isDisabled={hasOptionalComponent}
```

Comment:
does this `isDisabled` control whether the tooltip shows?

Just to be sure:

it should only show if there are no optional components

if there are optional components, the button should work with no tooltip

Reply:
Yup, isDisabled={hasOptionalComponent} would make the tooltip not show during an optional component (when button is not disabled).